result/metrics.yaml

```diff
@@ -51,3 +59,11 @@ components:
           type: array
           items:
             $ref: './bgpv6.yaml#/components/schemas/Bgpv6.Metric'
+        arp_entries:
+          description: >-
+            Arp entries - for resolved & unresolved.
```

Comment:
Arp entries - for resolved & unresolved ARP(s).

Reply:
Arp entries-> ARP entries

Reply:
Updated.